Error building on RPi 2 model B

When running `scons`on RPi 2 model B, I receive the following error:

`rpi_ws281x/ws2811.c:126 undefined reference to clock_gettime`

I've seen some suggestions on stack overflow to upgrade glibc to a newer version to fix it, but I'm curious to know if you have any other suggestions before I do that.  

Thanks!

--- a/SConstruct
+++ b/SConstruct
@@ -43,6 +43,7 @@ platforms = [
             'CPPPATH' : [
             ],
             'LINKFLAGS' : [
+                "-lrt",
             ],
         },
     ], 

